tests/functional/firefox/test_home.py

```diff
@@ -7,6 +7,9 @@
 from pages.firefox.home import FirefoxHomePage
 
 
+# /firefox Quantum page behind switch, unskip when the switch is removed
```

Comment:
I don't think we really need the comments here, as we have the `reason` referencing the bug.